### ISSUE TITLE
Map CNCF projects in the SAFE Landscape

### ISSUE THREAD

**Issue by ultrasaurus** [cn-security/safe#69]
TO DO:

- [ ] create draft categories: https://github.com/cn-security/safe/issues/66
- [ ] discuss in SAFE WG with initial categorization
- [ ] invite CNCF projects to review categories and sign up to meet with us to discuss their position in the landscape 
  - Security and Policy solutions
    - [ ] SPIFFE / SPIRE - Evan
    - [ ] Notary
    - [ ] TUF
    - [ ] OPA - Torin with updater
  - Projects with Security Concerns
    - [ ] gRPC
  - Platforms (consider sub-projects that can be used stand-alone, such as Envoy, Mixer):
    - [ ] Istio
    - [ ] Kubernetes

**Comment by izgeri** [comment]
@ultrasaurus I'd like to revise this issue slightly to clarify its purpose. Can we rename it to something like "Solicit feedback from relevant CNCF projects on SAFE landscape", and then remove  the first two bullets:
```
- [ ] create draft categories: #66
- [ ] discuss in SAFE WG with initial categorization
```

This focuses this particular issue on soliciting feedback once we've created draft categories in #66, which I think is beneficial.

I would also suggest that we create a third issue "Map projects from CNCF landscape into SAFE landscape".
This issue as currently written implies that only CNCF projects will be included in the SAFE landscape, but my understanding from the reference architecture group is that the security-focused landscape will be a part of the traditional CNCF landscape (in the same way that there is currently a separate "Serverless" landscape). In the current CNCF landscape, many projects besides official CNCF projects are included, though CNCF projects are highlighted. I would like to do the same on the SAFE landscape, which will require an effort to determine both which projects to include and where they belong on the SAFE landscape (work we have already begun, to some degree).

I'm not sure how we want to go about organizing issues that are linked together - I can create an epic for working on the landscape and make these 3 stories the children, or a milestone with these stories included. Please let me know if you have thoughts.

I am happy to participate in organizing these issues, etc, but might face difficulty due to my restricted privileges in the repo.

In sum:
- [ ] Can we revise this issue to be more focused on working with CNCF projects to get their feedback on our draft proposal?
- [ ] Can we add a separate issue for mapping security-related projects from the current CNCF landscape into the SAFE landscape?
- [ ] Does anyone have thoughts on how SAFE should manage linked stories in this repository? For now, it looks like we're relying on labels, but milestones or epics might be a nice addition.

**Comment by izgeri** [comment]
Or based on the governance PR, perhaps it makes sense to have this issue (and any additional related issues) all grouped into #66, to track the subproject with one issue.  Please advise.

**Comment by izgeri** [comment]
I've merged the content of this PR into #66. In particular, I've added a task breakdown of the work ahead in a comment there (https://github.com/cn-security/safe/issues/66#issuecomment-423642797) - @dshaw, can issue #69 please be closed now?

**Comment by dshaw** [comment]
Thank you, @izgeri
